shared/ui/Stream/OpenPullRequests.tsx

```diff
@@ -515,6 +518,18 @@ export const OpenPullRequests = React.memo((props: Props) => {
 				// Update default queries for users in a non-destructive way
 				if (derivedState.pullRequestQueries) {
 					patchQueries(defaultQueriesResponse, derivedState.pullRequestQueries, saveQueries);
+					let newObject = {};
+					//check if there is an empty array and if so, take it out
+					Object.keys(derivedState.pullRequestQueries).forEach(key => {
+						if (
+							derivedState.pullRequestQueries![key] &&
+							Array.isArray(derivedState.pullRequestQueries![key]) &&
+							derivedState.pullRequestQueries![key].length > 0
+						) {
```

Comment:
Can get rid of the ! with a null check

```suggestion
if (derivedState.pullRequestQueries &&
							derivedState.pullRequestQueries[key] &&
							Array.isArray(derivedState.pullRequestQueries[key]) &&
							derivedState.pullRequestQueries[key].length > 0
						) {
```